From 2847b6fa7c88e34916da19fbd46604f90b51023a Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Wed, 19 Dec 2012 23:50:23 -0500
Subject: [PATCH] Fix automagic dep on libsystemd-login

Just because it's installed doesn't mean we want to use it.
---
 configure.ac | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index ef0df5b..4da2055 100644
--- a/configure.ac
+++ b/configure.ac
@@ -177,17 +177,14 @@ if test "x$with_systemdsystemunitdir" != "xno"; then
 fi
 AM_CONDITIONAL(HAVE_SYSTEMD, [test -n "$systemdsystemunitdir"])
 
-PKG_CHECK_MODULES(LIBSYSTEMD_LOGIN,
-                  [libsystemd-login >= 44],
-                  have_libsystemd_login=yes,
-                  have_libsystemd_login=no)
-AM_CONDITIONAL(HAVE_LIBSYSTEMD_LOGIN, test x$have_libsystemd_login = xyes)
-if test "x$have_libsystemd_login" = "xyes"; then
+AC_ARG_ENABLE(libsystemd-login, AS_HELP_STRING([--enable-libsystemd-login],[Enable libsystemd-login support]),
+	      enable_libsystemd_login=$enableval, enable_libsystemd_login=yes)
+if test x$enable_libsystemd_login != xno; then
+	PKG_CHECK_MODULES(LIBSYSTEMD_LOGIN, [libsystemd-login >= 44])
 	AC_DEFINE([HAVE_LIBSYSTEMD_LOGIN], 1, [Define to 1 if libsystemd-login is available])
 fi
+AM_CONDITIONAL(HAVE_LIBSYSTEMD_LOGIN, test x$enable_libsystemd_login = xyes)
 AC_SUBST(HAVE_LIBSYSTEMD_LOGIN)
-AC_SUBST(LIBSYSTEMD_LOGIN_CFLAGS)
-AC_SUBST(LIBSYSTEMD_LOGIN_LIBS)
 
 dnl ---------------------------------------------------------------------------
 dnl - Where to install udev rules
-- 
1.8.0.2

